Fastfile

```diff
@@ -75,115 +73,113 @@ end
+  UI.message('Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined')
+  'enterprise'
 end
 
-def _update_team_id_if_necessary()
+def _update_team_id_if_necessary
   project_path = ENV['FL_PROJECT_SIGNING_PROJECT_PATH']
-  team_id = _get_team_id()
+  team_id = _get_team_id
   if !project_path.to_s.strip.empty? && !team_id.to_s.strip.empty?
     UI.message("Updating project team with project path '#{project_path}' and team id '#{team_id}'.")
     update_project_team(path: project_path, teamid: team_id)
   else
-    UI.message("Unable to find project path or project team so skipping updating project team.")
+    UI.message('Unable to find project path or project team so skipping updating project team.')
   end
 end
 
-def _get_team_id()
+def _get_team_id
   team_id = ENV['FL_PROJECT_TEAM_ID']
   unless team_id
-    UI.message("Attempting to extract team ID from `GYM_EXPORT_OPTIONS` since `FL_PROJECT_TEAM_ID` is not defined.")
-    team_id = get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: "teamID")
+    UI.message('Attempting to extract team ID from `GYM_EXPORT_OPTIONS` since `FL_PROJECT_TEAM_ID` is not defined.')
+    team_id = get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'],
+                                   key: 'teamID')
   end
   team_id
 end
 
-def _upload_to_hockey()
-  custom_notes = ENV['TAB_HOCKEY_RELEASE_NOTES'] || ""
-  notes = custom_notes == "" ? _create_change_log() : custom_notes
-  hockey(notes_type: "0", notes: notes)
+def _upload_to_hockey
+  custom_notes = ENV['TAB_HOCKEY_RELEASE_NOTES'] || ''
+  notes = custom_notes == '' ? _create_change_log : custom_notes
+  hockey(notes_type: '0', notes: notes)
 end
 
-def _create_change_log()
+def _create_change_log
   cmd = "git log --after={1.day.ago} --pretty=format:'%an%x09%h%x09%cd%x09%s' --date=relative"
   output = `#{cmd}`
-  return (output.length == 0) ? "No Changes" : output
+  output.length.empty? ? 'No Changes' : output
 end
 
-def _upload_to_test_flight()
-  pilot(username: ENV["ITUNES_CONNECT_USERNAME"],
-    team_id: ENV["ITUNES_CONNECT_TEAM_ID"],
-    itc_provider: ENV["ITUNES_CONNECT_PROVIDER"],
-    skip_waiting_for_build_processing: true,
-    skip_submission: true)
+def _upload_to_test_flight
+  pilot(username: ENV['ITUNES_CONNECT_USERNAME'],
+        team_id: ENV['ITUNES_CONNECT_TEAM_ID'],
+        itc_provider: ENV['ITUNES_CONNECT_PROVIDER'],
+        skip_waiting_for_build_processing: true,
+        skip_submission: true)
 end
 
-def _notify_slack()
-  if ENV['FL_SLACK_CHANNEL'].to_s.strip.empty?
-    return
-  end
+def _notify_slack
+  return if ENV['FL_SLACK_CHANNEL'].to_s.strip.empty?
   hockey_download_url = lane_context[SharedValues::HOCKEY_DOWNLOAD_LINK]
-  if hockey_download_url != nil
-    new_build_message = "A new build is available on <" + hockey_download_url + "|hockey>"
+  if !hockey_download_url.nil?
+    new_build_message = "A new build is available on <${hockey_download_url}|hockey>"
```

Comment:
Isn't it `#{ }` for string interpolation?

Reply:
You can tell I've been doing Groovy for the past few days, can't you? :P